### PR TITLE
update debian build

### PR DESCRIPTION
**Pull request by AlwinEsch**

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -9,11 +9,16 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
```
Comment (Rechi) [inline debian/rules]: unused

Comment (AlwinEsch) [reply debian/rules]: Wow you are fast! :-D

**Comment by AlwinEsch** [comment]
Is there something known about the windows build fail on Jenkins?

Found during googling this https://bugs.eclipse.org/bugs/show_bug.cgi?id=518765 which report a build crash on windows if value is "/Zi" and not come with "/Z7".

**Comment by AlwinEsch** [comment]
Forget it for here, had the addons confused, was on https://github.com/xbmc/screensaver.asteroids/pull/14

**Pull request merged**